leapp/libraries/stdlib/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""
+:py:mod:`leapp.libraries.stdlib` represents a location for functions that otherwise would be defined multiple times across leapp actors
```

Comment:
Keep the lines up to 120 chars. This one has 136.

Reply:
updated